src/components/views/rooms/RoomList.js

```diff
@@ -164,7 +164,12 @@ module.exports = React.createClass({
 
         // load stored sizes
         Object.entries(this.subListSizes).forEach(([id, size]) => {
-            this.resizer.forHandleWithId(id).resize(size);
+            const handle = this.resizer.forHandleWithId(id);
+            if (handle) {
+                try {
+                    handle.resize(size);
+                } catch(_e) {}
```

Comment:
do we need to log this?

Reply:
Good catch, it's not needed because I already [handle it closer to the source](https://github.com/matrix-org/matrix-react-sdk/pull/2341/commits/75925f6192e1d1a9934325a9dedfa26e9378bc26). Not sure what I was thinking :)

Reply:
And in the other location, I wouldn't log it, just keep the FIXME comment for now, as this is just patchwork to make community UX working again, but we'll definitely come back to it for the redesign.